### PR TITLE
Testing - mock setup for redis

### PR DESCRIPTION
**Pull request by sondrekje**
### Description

run tests against real redis or mock

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -59,6 +59,7 @@
     "@types/swagger-ui-express": "4.1.3",
     "@types/ua-parser-js": "0.7.36",
     "@types/uuid": "8.3.4",
+    "ioredis-mock": "^7.4.0",
```
Comment (Bruception) [inline backend/package.json:62]: save exact

File: backend/__tests__/tasks/george.spec.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const IS_USING_REDIS_MOCK = process.env.REDIS_URI?.includes("mock") ?? false;
+
+const describeIf = (condition: boolean): jest.Describe =>
+  condition ? describe : describe.skip;
+
+describeIf(!IS_USING_REDIS_MOCK)("George", () => {
+  it("should be able to initJobQueue", async () => {
+    await RedisClient.connect();
+
+    await initJobQueue(RedisClient.getConnection());
```
Comment (Bruception) [inline backend/__tests__/tasks/george.spec.ts:14]: initJobQueue is not async

File: backend/__tests__/tasks/george.spec.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+process.env.MODE = undefined; // Remove dev mode
```
Comment (Bruception) [inline backend/__tests__/tasks/george.spec.ts:1]: Does this interfere with other tests?

Comment (sondrekje) [reply backend/__tests__/tasks/george.spec.ts]: probably yeah. would have preferred to not run tests in dev mode at all, but if we are to do this we need to restore the old environment

File: backend/__tests__/tasks/george.spec.ts (hunk 2 above)
Comment (sondrekje) [reply backend/__tests__/tasks/george.spec.ts]: didn't notice that after rebasing, removed the entire test now.

**Review by Bruception — APPROVED** [review]
LGTM IIW

**Pull request merged**